cmd/assetsvc/postgresql_utils.go

```diff
@@ -56,12 +56,12 @@ func (m *postgresAssetManager) getPaginatedChartList(namespace, repo string, pag
 	clauses := []string{}
 	queryParams := []interface{}{}
 	if namespace != dbutils.AllNamespaces {
-		clauses = append(clauses, "repo_namespace = $1")
-		queryParams = append(queryParams, namespace)
+		queryParams = append(queryParams, namespace, m.GetKubeappsNamespace())
+		clauses = append(clauses, fmt.Sprintf("(repo_namespace = $%d OR repo_namespace = $%d)", len(queryParams)-1, len(queryParams)))
```

Comment:
this would be more readable if it were $1 and $2 but I understand why you are calculating those anyway

Reply:
Yeah, like I had it initially. Unfortunate given they may not be the first and second... ah, that's what you mean, that if these are included they will always be the first and second params, it's only really the following one which needs to be calculated... true. Updated :)